File: src/instruction/InstructionEntry.java
package instruction;

public class InstructionEntry {
    private String ins;
    private int op = -10010;

    public InstructionEntry(String ins) {
        this.ins = ins;
    }
    public InstructionEntry(String ins, int op){
        this.ins = ins;
        this.op = op;
    }

    public String getIns() {
        return ins;
    }

    public void setIns(String ins) {
        this.ins = ins;
    }

    public int getOp() {
        return op;
    }

    public void setOp(int op) {
        this.op = op;
    }

}

File: bin/instruction/instruction_entry.dart
class InstructionEntry {
  String ins;
  int op;

  InstructionEntry(this.ins, {this.op = -10010}) {
  }
  //   InstructionEntry(String ins, int op){
  //     this.ins = ins;
  //     this.op = op;
  // }

  String getIns() {
    return ins;
  }

  void setIns(String ins) {
    this.ins = ins;
  }

  int getOp() {
    return op;
  }

  void setOp(int op) {
    this.op = op;
  }
}
